plugins/GrapesJsBuilderBundle/Assets/library/js/grapesjs-editor.css

```diff
@@ -0,0 +1,3 @@
+html, body, [data-gjs-type=wrapper] {
+    height: auto !important;
```

Comment:
Is this necessary? It affect everything.

Reply:
@volha-pivavarchyk could you respond to the question here? Thanks!

Reply:
@escopecz I added this style only into the builder iframe. It affects the builder, and that's what the target is.

Reply:
If it is enclosed in an iframe than it's ok. Thanks for explanation!